met-api/tests/unit/api/test_survey.py

```diff
@@ -28,12 +28,17 @@
 from tests.utilities.factory_utils import (
     factory_auth_header, factory_engagement_model, factory_survey_model, factory_tenant_model, set_global_tenant)
 
+surveys_url = '/api/surveys/'
```

Comment:
can you add a few more tests for clone? 
If we don't have them already

Reply:
Added a clone survey test in the api test